server/tests/publish/belga_newsml_1_2_items_chain_tests.py

```diff
@@ -680,6 +680,8 @@ def test_belga_text_newscomponent(self):
             ('urn:belga.be:coverage:6690595', 'nl'),
             ('urn:belga.be:360archive:77777777', 'nl'),
             ('original-fr', 'fr'),
+            ('urn:belga.be:image:154620545', 'fr'),
+            ('urn:belga.be:image:154670415', 'fr'),
```

Comment:
imo here the language of those images should be nl, that's not changing

Reply:
here we already have for NL https://github.com/superdesk/superdesk-belga/blob/develop/server/tests/publish/belga_newsml_1_2_items_chain_tests.py#L678-#L679, I think we are changing for NL to FR so we need fr also right?

Reply:
when we translate text with pictures we don't translate the media inside, do we?

Reply:
I just revert those changes of SDBELGA-514 because as per the new changes and before SDBELGA-514 also it is working the same. 
and as far as I know, we are not translating pictures (not sure) but after translation when we publish the main item then in the output XML we have two picture items as I have shared in the previous comment(https://github.com/superdesk/superdesk-belga/pull/375#discussion_r780089322) so we need those old changes here.